frameworks/CSharp/aspnetcore/Benchmarks/Benchmarks.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```

Comment:
Global usings were made to be not "global breaking" in RC1 - they're opt-in instead of opt-out now.

This line is shouldn't be needed.

Reply:
Thanks, updated. It was added during preview-7, I assume at that time it was breaking.